Documentation/ceph-prerequisites.md

```diff
@@ -11,7 +11,7 @@ To make sure you have a Kubernetes cluster that is ready for `Rook`, review the
 In order to configure the Ceph storage cluster, at least one of these local storage options are required:
 - Raw devices (no partitions or formatted filesystems)
 - Raw partitions (no formatted filesystem)
-- PVs available from a storage class in `block` mode
+- PVs available from a storage class in `block` mode. These PVs should be backed by the above mentioned options or Logical Volumes.
```

Comment:
Are there PVs in block mode that won't work? I would hope that the backing type doesn't matter. If it's a block mode PV hopefully it will work with rook. But if something doesn't work, I'd suggest we give examples of those, rather than saying the options that do work.

Reply:
I thought the devices other devices than raw partition, and LV are denied by ceph-volume. However, I'll remove this change from the next push. I reread the ceph-volume's source and found that I would misunderstand something. I'll check again to see if my understanding is correct and It'll take some time.